account/generator/account.go

```diff
@@ -48,7 +52,8 @@ type AccountInfo struct {
 // IdentifiedAccountInfo contains AccountInfo and the ID of an account.
 type IdentifiedAccountInfo struct {
 	AccountInfo
-	ID string `json:"id"`
+	ID     string `json:"id"`
```

Comment:
Let's explain what it is and how `KeyUID` (maybe rename to `KeyID`) is different from `ID`.

Reply:
We can rename it to `KeyID` but keep it as `keyUid` in json, would that work for you?